ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/SearchTree.java

```diff
@@ -92,6 +92,10 @@ void setAvailableRangeActions(Set<RangeAction> rangeActions) {
         availableRangeActions = rangeActions;
     }
 
+    void setPrePerimeterRangeActionSetPoints(Map<RangeAction, Double> prePerimeterRangeActionSetPoints) {
+        this.prePerimeterRangeActionSetPoints = prePerimeterRangeActionSetPoints;
+    }
+
```

Comment:
I'm not a fan of exposing this method to the whole package (even though you only need it for tests).
can you use mocks in the unit tests instead ?

Reply:
I'm not a fan either. Sadly I didn't find a way with mocks... I believe we had the same problem with the tree parameters and the available range actions.